dev-tools/mage/fmt.go

```diff
@@ -50,13 +50,10 @@ func Format() {
 	mg.Deps(GoImports, PythonAutopep8)
 }
 
-// GoImports executes goimports against all .go files in and below the CWD. It
-// ignores vendor/ and generator/_templates/ directories.
+// GoImports executes goimports against all .go files in and below the CWD.
 func GoImports() error {
 	goFiles, err := FindFilesRecursive(func(path string, _ os.FileInfo) bool {
-		return filepath.Ext(path) == ".go" &&
-			!strings.Contains(path, "vendor/") &&
```

Comment:
Nit. Should we keep vendor in case someone works locally with a vendor directory? Though no idea if this currently works well.

Reply:
It is not supported anymore in Beats.